MediaAdmin/FileUtils/Image/ImagickImageManager.php

```diff
@@ -70,41 +70,44 @@ protected function resizeImage(array $format, Imagick $image)
 
     /**
      * Resize an image keeping its ratio to the height $height
-     * 
+     *
      * @param Imagick $image
      * @param int     $height
-     * 
+     * @param bool    $setUpMinWidth
+     *
      * @return Imagick
      */
-    protected function resizeOnHeight(Imagick $image, $height)
+    protected function resizeOnHeight(Imagick $image, $height, $setUpMinWidth = false)
```

Comment:
method resizeOnHeight and resizeOnWidth are almost similar, i think you can  factorize